OpenRA.Mods.Common/Widgets/Logic/Ingame/ObserverStatsLogic.cs

```diff
@@ -344,10 +344,11 @@ ScrollItemWidget EconomyStats(Player player)
 
 			var res = player.PlayerActor.Trait<PlayerResources>();
 			var stats = player.PlayerActor.TraitOrDefault<PlayerStatistics>();
-			if (stats == null) return template;
+			if (stats == null)
+				return template;
 
 			template.Get<LabelWidget>("CASH").GetText = () => "$" + (res.Cash + res.Resources);
```

Comment:
We should probably wrap all these in `CachedTransform`s to reduce string allocations...